cmd/kubeadm/app/preflight/checks.go

```diff
@@ -52,6 +53,10 @@ const (
 	externalEtcdRequestTimeout  = time.Duration(10 * time.Second)
 	externalEtcdRequestRetries  = 3
 	externalEtcdRequestInterval = time.Duration(5 * time.Second)
+
+	// NB. in semver patches number is a numeric, while prerelease is a string where numeric identifiers always have lower precedence than non-numeric identifiers.
+	//     thus setting the value to x.y.0-0 we are defining the very first patch - prereleases within x.y minor release.
+	firstPatchAndPrelease = "0-0"
```

Comment:
Because I want to give a warning when the user asks for v1.X.* (including also pre-releases) 

With v1.X.0-0 I get this behaviour, while with v1.X.0 (without the final -0) I will accept pre-releases.